spec/features/categories/index_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.feature 'Categories Index Page - Viewing Transactions', type: :feature do
+  scenario 'User can view category transactions' do
+    user = create_and_sign_in_user
+    category = create_category_for_user(user)
```

Comment:
- [ ] The below 4 tests are failing, Please make sure that all tests are passed without any errors.

![image](https://github.com/NoelLincoln/budget-app/assets/83803414/698b1f06-b22f-4f93-972a-4d8a6b137374)